packages/api/src/utils/parser.ts

```diff
@@ -621,3 +621,18 @@ export const parseEmailAddress = (from: string): addressparser.EmailAddress => {
   }
   return { name: '', address: from }
 }
+
+export const fetchFavicon = async (
+  url: string
+): Promise<string | undefined> => {
+  try {
+    // get the correct url if it's a redirect
+    const response = await axios.head(url)
```

Comment:
Do we know what the default axios timeouts are? Might want to make sure its kind of short for these.

Reply:
The default is `0`. I better put a `timeout: 1000` (1s) here

Reply:
Maybe `5s` is better if it is slow